presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -160,6 +163,32 @@ public void testArrayElements()
         assertInvalidFunction("CAST(ARRAY ['puppies', 'kittens'] AS ARRAY<BIGINT>)", INVALID_CAST_ARGUMENT);
     }
 
+    private static String generateLongSring(int size, char c)
```

Comment:
Use `Strings.repeat("x", size)` in Guava. You don't need this `generateLongString`.